### PR TITLE
release 1.1.0

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag 

### DESCRIPTION
- release 1.1.0
- Fixed coveralls badge
- added pypi, coveralls, travis support

### CODE REVIEW

File: .gitchangelog.rc [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+##
+##   ACTION is one of 'chg', 'fix', 'new'
+##
+##       Is WHAT the change is about.
+##
+##       'chg' is for refactor, small improvement, cosmetic changes...
+##       'fix' is for bug fixes
+##       'new' is for new features, big improvement
+##
+##   SUBJECT is optional and one of 'dev', 'usr', 'pkg', 'test', 'doc'
```
Comment (flopezag) [inline .gitchangelog.rc]: SUBJECT or AUDIENCE?

Comment (geonexus) [inline .gitchangelog.rc]: It is the name provided by default by gitconfigrc developers, it should be an error.

Comment (flopezag) [inline .gitchangelog.rc]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**